lib/internal/debugger/inspect_client.js

```diff
@@ -297,7 +297,8 @@ class Client extends EventEmitter {
 
   async _discoverWebsocketPath() {
     const { 0: { webSocketDebuggerUrl } } = await this._fetchJSON('/json');
-    return URL.parse(webSocketDebuggerUrl).path;
+    const { pathname, search } = new URL(webSocketDebuggerUrl);
+    return `${pathname}${search}`;
```

Comment:
if this works then it's fine, but is there no internal URL.parse method?

Reply:
Correct. `url.parse()` is also deprecated by DEP0169 and should never be used in core either way.

Reply:
`URL.parse` is different, isn't it? or was this using `url.parse` before

Reply:
This was using [`url.parse`](https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost) before this PR.

Reply:
aha, that explains it, thanks :-)